docs/cloud/dbaas/backups.md

```diff
@@ -8,12 +8,12 @@
 All databases in DBaaS get automatically backed up about every 24 hours. Users can also manually
 backup their instances but that means that the next automatic backup will be taken about 24 hours
 after the last backup. All backups are stored for 90 days after which they get automatically
-deleted (not yet implemented). In the future, it will only be possible to delete backups automatically,
-users will not be able to manually delete their own backups.
+deleted, backups can not be manually delete by the user.
```

Comment:
... can not be manually deleted by the user. (d's missing)

Reply:
Fixed, I should not be allowed to use a keyboard.